index.html

```diff
@@ -40,9 +40,141 @@ <h4 class="sub-heading">
+    </div>
+    <button class="btn">See project</button>
+    <div class="block2">
+       <h1>Profesional Art <br>
+        Printing Data</h1>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <span class="card-item">
+            <div>HTML</div>
+            <div>Bootstrap</div>
+            <div>Ruby</div>
+        </span>
+        <button class="btn-2">See project</button>
+    </div>
+    <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
+         <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+         </p>
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
+         <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+         </p>
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
+         <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+         </p>
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
+         <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+         </p>
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
+         <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+         </p>
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="banner-2">
+        <div class="main-1">
+            <h1 class="heading-2">
+                About me
+            </h1>
```

Comment:
Please remove all the exciding h1 heading in this page as It is allow to only have one h1 tag per page please consider refractoring your code use other heading tags: h2,h3,h4,h5,h6